### PR TITLE
remove Transifex calls for OEP-58

### PR DESCRIPTION
**Pull request by hopewise**
## Breaking change!

This change breaks the Jenkins transifex integration which has been deprecated in favor of the new GitHub Transifex App integration as part of [OEP-58](https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0058-arch-translations-management.html#specification).

## Changes

 - Removes all direct use of `tx pull` and `tx push` commands from the micro-frontend in favor
of the [`atlas pull`](https://github.com/openedx/openedx-atlas/) command.
 - Remove source and language translations from the repositories, hence no `.json` files will be committed into the repos. 
 - `src/i18n/index.js` should export and empty array so the `make pull_translations` override it with the dynamic list of languages.
 - Remove the experimental `OPENEDX_ATLAS_PULL` flag to make `atlas pull` the default. 
 - Remove all Transifex related `Makefile` targets and other files.

Test results
------------

 - [ ] Verify that `make pull_translations` works as expected.

<details><summary>make pull_translations test results</summary>

```
# I've run the following commands:
$ make requirements
$ make pull_translations
$ git diff

# Output of the commmands:

$ make requirements
make requirements
npm ci
npm WARN prepare removing existing node_modules/ before installation

> core-js@2.6.12 postinstall /Users/sadaftech/Documents/zeit-labs/frontend-app-account/node_modules/babel-runtime/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"


> core-js@3.32.2 postinstall /Users/sadaftech/Documents/zeit-labs/frontend-app-account/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"


> @fortawesome/fontawesome-common-types@0.2.36 postinstall /Users/sadaftech/Documents/zeit-labs/frontend-app-account/node_modules/@fortawesome/free-solid-svg-icons/node_modules/@fortawesome/fontawesome-common-types
> node attribution.js

Font Awesome Free 0.2.36 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)


> @fortawesome/free-solid-svg-icons@5.15.4 postinstall /Users/sadaftech/Documents/zeit-labs/frontend-app-account/node_modules/@fortawesome/free-solid-svg-icons
> node attribution.js

Font Awesome Free 5.15.4 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)


> @fortawesome/fontawesome-common-types@0.2.36 postinstall /Users/sadaftech/Documents/zeit-labs/frontend-app-account/node_modules/@fortawesome/free-regular-svg-icons/node_modules/@fortawesome/fontawesome-common-types
> node attribution.js

Font Awesome Free 0.2.36 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)


> @fortawesome/free-regular-svg-icons@5.15.4 postinstall /Users/sadaftech/Documents/zeit-labs/frontend-app-account/node_modules/@fortawesome/free-regular-svg-icons
> node attribution.js

Font Awesome Free 5.15.4 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)


> @fortawesome/fontawesome-common-types@0.2.36 postinstall /Users/sadaftech/Documents/zeit-labs/frontend-app-account/node_modules/@fortawesome/free-brands-svg-icons/node_modules/@fortawesome/fontawesome-common-types
> node attribution.js

Font Awesome Free 0.2.36 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)


> @fortawesome/free-brands-svg-icons@5.15.4 postinstall /Users/sadaftech/Documents/zeit-labs/frontend-app-account/node_modules/@fortawesome/free-brands-svg-icons
> node attribution.js

Font Awesome Free 5.15.4 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)


> @fortawesome/fontawesome-common-types@0.2.36 postinstall /Users/sadaftech/Documents/zeit-labs/frontend-app-account/node_modules/@fortawesome/fontawesome-svg-core/node_modules/@fortawesome/fontawesome-common-types
> node attribution.js

Font Awesome Free 0.2.36 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)


> @fortawesome/fontawesome-svg-core@1.2.36 postinstall /Users/sadaftech/Documents/zeit-labs/frontend-app-account/node_modules/@fortawesome/fontawesome-svg-core
> node attribution.js

Font Awesome Free 1.2.36 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)


> @fortawesome/fontawesome-common-types@6.4.2 postinstall /Users/sadaftech/Documents/zeit-labs/frontend-app-account/node_modules/@fortawesome/fontawesome-common-types
> node attribution.js

Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)
Copyright 2023 Fonticons, Inc.


> @fortawesome/fontawesome-svg-core@6.4.2 postinstall /Users/sadaftech/Documents/zeit-labs/frontend-app-account/node_modules/@edx/paragon/node_modules/@fortawesome/fontawesome-svg-core
> node attribution.js

Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)
Copyright 2023 Fonticons, Inc.


> @fortawesome/fontawesome-svg-core@6.4.2 postinstall /Users/sadaftech/Documents/zeit-labs/frontend-app-account/node_modules/@edx/frontend-component-header/node_modules/@fortawesome/fontawesome-svg-core
> node attribution.js

Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)
Copyright 2023 Fonticons, Inc.


> core-js@2.6.12 postinstall /Users/sadaftech/Documents/zeit-labs/frontend-app-account/node_modules/babel-polyfill/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"


> @fortawesome/free-solid-svg-icons@6.4.2 postinstall /Users/sadaftech/Documents/zeit-labs/frontend-app-account/node_modules/@edx/frontend-component-header/node_modules/@fortawesome/free-solid-svg-icons
> node attribution.js

Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)
Copyright 2023 Fonticons, Inc.


> @fortawesome/free-regular-svg-icons@6.4.2 postinstall /Users/sadaftech/Documents/zeit-labs/frontend-app-account/node_modules/@edx/frontend-component-header/node_modules/@fortawesome/free-regular-svg-icons
> node attribution.js

Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)
Copyright 2023 Fonticons, Inc.


> @fortawesome/free-brands-svg-icons@6.4.2 postinstall /Users/sadaftech/Documents/zeit-labs/frontend-app-account/node_modules/@edx/frontend-component-header/node_modules/@fortawesome/free-brands-svg-icons
> node attribution.js

Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)
Copyright 2023 Fonticons, Inc.


> @fortawesome/fontawesome-svg-core@6.4.2 postinstall /Users/sadaftech/Documents/zeit-labs/frontend-app-account/node_modules/@edx/frontend-component-footer/node_modules/@fortawesome/fontawesome-svg-core
> node attribution.js

Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)
Copyright 2023 Fonticons, Inc.


> @fortawesome/free-solid-svg-icons@6.4.2 postinstall /Users/sadaftech/Documents/zeit-labs/frontend-app-account/node_modules/@edx/frontend-component-footer/node_modules/@fortawesome/free-solid-svg-icons
> node attribution.js

Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)
Copyright 2023 Fonticons, Inc.


> @fortawesome/free-regular-svg-icons@6.4.2 postinstall /Users/sadaftech/Documents/zeit-labs/frontend-app-account/node_modules/@edx/frontend-component-footer/node_modules/@fortawesome/free-regular-svg-icons
> node attribution.js

Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)
Copyright 2023 Fonticons, Inc.


> @fortawesome/free-brands-svg-icons@6.4.2 postinstall /Users/sadaftech/Documents/zeit-labs/frontend-app-account/node_modules/@edx/frontend-component-footer/node_modules/@fortawesome/free-brands-svg-icons
> node attribution.js

Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)
Copyright 2023 Fonticons, Inc.


> sharp@0.32.5 install /Users/sadaftech/Documents/zeit-labs/frontend-app-account/node_modules/sharp
> (node install/libvips && node install/dll-copy && prebuild-install) || (node install/can-compile && node-gyp rebuild && node install/dll-copy)

sharp: Using cached /Users/sadaftech/.npm/_cacache/_libvips/libvips-8.14.4-darwin-x64.tar.br
sharp: Integrity check passed for darwin-x64

> core-js-pure@3.32.2 postinstall /Users/sadaftech/Documents/zeit-labs/frontend-app-account/node_modules/core-js-pure
> node -e "try{require('./postinstall')}catch(e){}"

added 1760 packages in 49.235s

make pull_translations
rm -rf src/i18n/messages
mkdir src/i18n/messages
cd src/i18n/messages \
      && atlas pull \
               translations/paragon/src/i18n/messages:paragon \
               translations/frontend-component-footer/src/i18n/messages:frontend-component-footer \
               translations/frontend-component-header/src/i18n/messages:frontend-component-header \
               translations/frontend-app-account/src/i18n/messages:frontend-app-account
Pulling translation files
 - directory: translations/paragon/src/i18n/messages:paragon translations/frontend-component-footer/src/i18n/messages:frontend-component-footer translations/frontend-component-header/src/i18n/messages:frontend-component-header translations/frontend-app-account/src/i18n/messages:frontend-app-account
 - repository: openedx/openedx-translations
 - branch: main
 - filter: Not Specified
 - expand-glob: Not Specified
Creating a temporary Git repository to pull translations into "./translations_TEMP"...
Done.
Setting git sparse-checkout rules...
Done.
Pulling translation files from the repository...
remote: Enumerating objects: 12, done.
remote: Counting objects: 100% (12/12), done.
remote: Compressing objects: 100% (9/9), done.
remote: Total 12 (delta 5), reused 8 (delta 3), pack-reused 0
Receiving objects: 100% (12/12), 35.54 KiB | 438.00 KiB/s, done.
Resolving deltas: 100% (5/5), done.
Your branch is up to date with 'origin/main'.
Done.
Copying translations from "./translations_TEMP/translations/paragon/src/i18n/messages" to "./paragon"...
Done.
Copying translations from "./translations_TEMP/translations/frontend-component-footer/src/i18n/messages" to "./frontend-component-footer"...
Done.
Copying translations from "./translations_TEMP/translations/frontend-component-header/src/i18n/messages" to "./frontend-component-header"...
Done.
Copying translations from "./translations_TEMP/translations/frontend-app-account/src/i18n/messages" to "./frontend-app-account"...
Done.
Removing temporary directory...
Done.

Translations pulled successfully!
./node_modules/.bin/intl-imports.js paragon frontend-component-header frontend-component-footer frontend-app-account



```

</details> 

Merge timeline
-----------------------

This should only be merged after [Translation Infrastructure update OEP-58](https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0058-arch-translations-management.html#specification) is fully implemented.

The timing announcement will be shared by @brian-smith-tcril on [#translations-project-fc-0012](https://openedx.slack.com/archives/C04R6TUJB7T) Open edX Slack channel.

Keep this pull request as a draft to prevent accidental merge.

### Pre-merge checklist

 - [ ] Wait for approval on https://github.com/openedx/wg-translations/issues/20

References
----------

This contribution is part of the [FC-0012 project](https://openedx.atlassian.net/l/cp/XGS0iCcQ) which is sparked by the [Translation Infrastructure update OEP-58](https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0058-arch-translations-management.html#specification).

Up-to-date project overview and details are available in the [Approach Memo and Technical Discovery: Translations Infrastructure Implementation](https://docs.google.com/document/d/11dFBCnbdHiCEdZp3pZeHdeH8m7Glla-XbIin7cnIOzU/edit#) document.

Join the conversation on [Open edX Slack #translations-project-fc-0012](https://openedx.slack.com/archives/C04R6TUJB7T).

Check the links above for full information about the overall project.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @hopewise!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.




:warning: We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](https://openedx.org/cla)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/openedx/.github/blob/master/CONTRIBUTING.md)     file for more information.     If you've signed an agreement in the past, you may need to re-sign.  See     [The New Home of the Open edX Codebase](https://open.edx.org/blog/the-new-home-of-the-open-edx-codebase/)     for details.

    
Once you've signed the CLA, please allow 1 business day for it to be processed.  After this time, you can re-run the     CLA check by adding a comment here that you have signed it.  If the problem persists, you can tag the `@openedx/cla-problems` team in a     comment on your PR for further assistance.

File: Makefile [hunk 1]
```diff
@@ -40,28 +38,12 @@ detect_changed_source_translations:
...
-	$(transifex_utils) $(transifex_temp) --comments --v3-scripts-path
-	# Pushing comments to Transifex...
-	./node_modules/@edx/reactifex/bash_scripts/put_comments_v3.sh
-
-ifeq ($(OPENEDX_ATLAS_PULL),)
-# Pulls translations from Transifex.
-pull_translations:
-	tx pull -t -f --mode reviewed --languages=$(transifex_langs)
-else
 # Experimental: OEP-58 Pulls translations using atlas
```
Comment (OmarIthawi) [inline Makefile:41]: Please remove this comment. 

When this pull request is merged, OEP-58 is no longer experimental.
```suggestion
```

**Review by OmarIthawi — COMMENTED** [review]
Thanks Samir! One comment.

File: Makefile [hunk 2]
```diff
@@ -1,6 +1,4 @@
 export TRANSIFEX_RESOURCE = frontend-app-account
```
Comment (OmarIthawi) [inline Makefile:1]:
```suggestion
```

This is no longer needed.

File: Makefile (hunk 1 above)
Comment (hopewise) [reply Makefile]: Done

File: Makefile (hunk 2 above)
Comment (hopewise) [reply Makefile]: Done

**Comment by OmarIthawi** [comment]
Thanks @hopewise. We need to add one more item before we can conclude this PR:

 - Run `make pull_translations` locally and post a git diff. I've updated the template so you can copy the `Test results` section from within: https://github.com/Zeit-Labs/fc-0012-pr-template/blob/main/mfe-deprecate-tx.md?plain=1#L14

**Comment by OmarIthawi** [comment]
Thanks for posting the test results @hopewise: https://gist.github.com/hopewise/9b395e29af53a97e3724f48a76603394
One more change.

The commit message should have the prefix `feat!: `. So it should become `feat! remove Transifex calls for OEP-58`. Similar to the https://github.com/openedx/frontend-app-gradebook/pull/353.

**Review by OmarIthawi — APPROVED** [review]
Looks great. Thanks @hopewise.

Let's wait for the https://github.com/openedx/wg-translations/issues/20 to be approved before making this as ready for merge.

**Comment by OmarIthawi** [comment]
@itsjeyd @timmc-edx @brian-smith-tcril this one is ready for merge.

**Review by brian-smith-tcril — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@hopewise 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-account&entry.867055334=https://github.com/openedx/frontend-app-account/pull/887&entry.1484655318=https://github.com/hopewise&entry.752974735=2023-09-29+09:07&entry.1917517419=2024-03-18+19:05&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**